psycopg3/psycopg3/connection.py

```diff
@@ -448,15 +448,20 @@ def close(self) -> None:
         """Close the database connection."""
         self.pgconn.finish()
 
-    def cursor(self, name: str = "", binary: bool = False) -> "Cursor":
+    def cursor(
+        self,
+        name: str = "",
+        binary: bool = False,
+        row_factory: RowFactory = cursor.default_row_factory,
```

Comment:
I change the default to `None` in commit "Make row factory optional"; I'll add the attribute to connection later on.

Reply:
> I think it would be good to have the row factory an attribute of the connection too, which would provide a connection default as a preference of the global default.

Done in f99461c835cb3c2effcd81042b0d98945201cb72.

Reply:
Hm. The way it's done is not quite right, I'll rework that.

Reply:
Fixed in f4e874ca04acec37a37e46b9b6a228a463c0cf95 (replacing f99461c835cb3c2effcd81042b0d98945201cb72), hopefully.